src/components/v5/common/ActionSidebar/partials/MultiSigSidebar/utils.ts

```diff
@@ -0,0 +1,31 @@
+import { ColonyActionType } from '~gql';
+import { type ColonyAction } from '~types/graphql.ts';
+import { updateContributorVerifiedStatus } from '~utils/members.ts';
+
+export const handleMultiSigFinalized = (action: ColonyAction) => {
```

Comment:
do we maybe want to extract this into a helper called `handleMotionCompleted` and reuse it between multi sig finalization and motion finalization?